app/src/main/java/org/thoughtcrime/securesms/trustedIntroductions/jobs/TrustedIntroductionsReceiveJob.java

```diff
@@ -49,31 +49,31 @@ public class TrustedIntroductionsReceiveJob extends BaseJob {
   private static final String KEY_BODY_PARSED = "bodyParsed";
   private static final String KEY_INTRODUCTIONS = "serialized_remaining_introduction_data";
 
-  public TrustedIntroductionsReceiveJob(@NonNull RecipientId introducerId, @NonNull String messageBody, @NonNull long timestamp){
-    this(introducerId,
+  public TrustedIntroductionsReceiveJob(@NonNull String messageBody, @NonNull long timestamp){
```

Comment:
Why it became null? 

Line 66: introducerId before was NonNull and now is Nullable, why so? Can it be null?